test/integration/infra.go

```diff
@@ -391,7 +391,7 @@ func (infra *infra) applyConfig(inFile string, data map[string]string) error {
 	}
 
 	glog.Info("Sleeping for the config to propagate")
-	time.Sleep(3 * time.Second)
+	time.Sleep(20 * time.Second)
```

Comment:
don't bundle this here. i have a fix in another PR. just wait for cluster to quiet down to pass tests.

Reply:
I cannot wait...

Reply:
If your PR gets in faster, it will ask me to merge and it will disappear. What value do you use?

Reply:
Yes you merged this? Really?